lib/ansible/modules/source_control/github_webhook.py

```diff
@@ -0,0 +1,240 @@
+#!/usr/bin/python
+#
+# Copyright: (c) Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: github_webhook
+short_description: Manage Github webhooks
+version_added: "2.6"
+description:
+  - "Create and delete Github webhooks"
+requirements:
+  - "PyGithub >= 1.3.5"
+options:
+  repository:
+    description:
+      - Full name of the repository to configure a hook for
+    required: true
+  url:
+    description:
+      - URL to which payloads will be delivered
```

Comment:
Add description that this different that `github_url` parameter ?

Reply:
I feel like this is pretty obvious. I'm not sure how to clarify it beyond the descriptions that are already in place.